README.md

```diff
@@ -8,20 +8,18 @@ Our pipeline also provides utilities to export the models to ONNX format for use
 We will grow the collection of the models suitable for high-throughput imaging and phenotyping.
 Expect rough edges until we release a PyPI package.
 
-![virtual_staining](docs/figures/phase_to_nuclei_membrane.svg)
+![virtual_staining](https://github.com/mehta-lab/VisCy/blob/main/docs/figures/phase_to_nuclei_membrane.svg?raw=true)
 
 This pipeline evolved from the [TensorFlow version of virtual staining pipeline](https://github.com/mehta-lab/microDL), which we reported in [this paper in 2020](https://elifesciences.org/articles/55502). The previous pipeline is now a public archive, and we will be focusing our efforts on VisCy.
```

Comment:
I would move this comment by the citation of the life paper or remove it. I don't think this links relevant.

Reply:
Removed.